.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        ruby-version: [2.2.10, 2.3.8, 2.4.10, 2.5.9, 2.6.8, 2.7.4, 3.0.2, jruby-9.2.17.0]
+        ruby-version: [2.2.10, 2.3.8, 2.4.10, 2.5.9, 2.6.8, 2.7.4, 3.0.2, jruby-9.2.19.0]
```

Comment:
@amhuntsman - Though this change is also in httprb PR #722, we wanted to add it here to see if it helped with some odd test failures that appear on jruby's rack multiverse tests, but nowhere else.

Reply:
It didn't hurt those failures, so we'd like to keep this change in the PR if it doesn't create too much of a conflict.